src/main/kotlin/com/exactpro/th2/common/schema/strategy/fieldExtraction/impl/AnyMessageFieldExtractionStrategy.kt

```diff
@@ -17,14 +17,30 @@
 package com.exactpro.th2.common.schema.strategy.fieldExtraction.impl
 
 import com.exactpro.th2.common.grpc.AnyMessage
-import com.exactpro.th2.common.grpc.Message
 import com.exactpro.th2.common.schema.message.toJson
 import com.exactpro.th2.common.schema.strategy.fieldExtraction.AbstractTh2MsgFieldExtraction
+import com.exactpro.th2.common.schema.strategy.fieldExtraction.FieldExtractionStrategy
 
-class AnyMessageFieldExtractionStrategy : AbstractTh2MsgFieldExtraction() {
-    override fun parseMessage(message: com.google.protobuf.Message): Message {
+class AnyMessageFieldExtractionStrategy : FieldExtractionStrategy {
```

Comment:
Change to `FiledExtractionStrategy` because `AbstractTh2MsgFieldExtraction` requires override method `parsedMessage` which return parsed message.